server/app/controllers/applicant/ApplicantProgramReviewController.java

```diff
@@ -134,7 +134,11 @@ public CompletionStage<Result> review(Request request, long applicantId, long pr
                                 .programBySlug(
                                     request.flash().get("redirected-from-program-slug").get())
                                 .url(),
-                            MessageKey.INITIAL_LOGIN_MODAL_PROMPT,
+                            messages.at(
+                                MessageKey.INITIAL_LOGIN_MODAL_PROMPT.getKeyName(),
+                                // The applicant portal name should always be set (there is a
+                                // default setting as well).
+                                settingsManifest.getApplicantPortalName().orElse("PortalMissing")),
```

Comment:
**suggestion (blocking):** a more graceful degradation if the config value is missing would be to use an empty string so the UI reads "log in to your account" rather than "log in to your PortalMissing account"

Reply:
Went ahead and committed the change, but I'm not sure this makes sense in other languages where the subject-object-verb syntax doesn't follow English?